### ISSUE TITLE
Can this be used with Express?

### ISSUE THREAD

**Issue by alsoicode** [insin/newforms#56]
I'm a Django developer most of my day, but I'm starting to do more and more on the Node side. Can newforms be used with Express?

**Comment by insin** [comment]
Yes, BUT I haven't tried using it with string template engines since I switched it over to outputting ReactElements. You might have to write some helpers to do a React.renderToStaticMarkup step if you're not intending to render with React on the server.

This repo uses the last pre-React version (v0.4.2) with Jade mixins for form rendering, for example: https://github.com/insin/successcrm

Revisiting the string template use case is on the TODO list.

**Comment by alsoicode** [comment]
Thank you for the quick answer. I'll take a look at that pre-React version as soon as I can. It would be super if the library could be used with multiple templating engines like Jade, Nunjucks, etc.
